src/main/java/pl/pawit/Questions.java

```diff
@@ -0,0 +1,51 @@
+package pl.pawit;
+
+
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class Questions {
```

Comment:
nazwa klasy mi nie pasuje. Proponuje podzial:
Klasa np. FileHandler -> tam zrobisz metode do odczytu pytan i metode do odczytu odpowidzi
klase QuestionHandler -> tam zrobisz metode ktora wywola metode do czytania pliku z FileHandlera i przygotuje i wybierze meotde do przeczytania
klase AnswerHandler -> Bedzie obslugiwala odpowiedz od użytkownika 

Pozniej ewentualnie bardziej sie to przemysli ale na ten moment bym tak zrobil